meeting.go

```diff
@@ -136,7 +136,7 @@ type (
 		EncryptedPassword string        `json:"encrypted_password"`
 		// PMI is Personal Meeting ID. Only used for scheduled meetings and recurring meetings with
 		// no fixed time
-		PMI            int64           `json:"pmi"`
+		PMI            string          `json:"pmi"`
```

Comment:
This change does not look correct

<img width="655" alt="Screen Shot 2021-05-07 at 10 39 41 AM" src="https://user-images.githubusercontent.com/1058475/117487908-8d567580-af20-11eb-9df5-e360df8dab1e.png">